src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx

```diff
@@ -63,21 +62,18 @@ const PlanDefinitionList = (props: PlanListProps) => {
   /** async function to load the data */
   async function loadData() {
     try {
-      setLoading(plans.length < 1); // only set loading when there are no plans
       const planObjects = await apiService.list();
       fetchPlans(planObjects);
     } catch (e) {
       displayError(e);
-    } finally {
-      setLoading(false);
     }
   }
 
   useEffect(() => {
     loadData().catch(err => displayError(err));
   }, []);
 
-  if (loading === true) {
+  if (plans.length < 1) {
```

Comment:
If we get 0 plans from the API, does this mean we will show the loading indicator forever?